components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/gen/java/org/wso2/carbon/identity/api/server/configs/v1/ConfigsApi.java

```diff
@@ -120,6 +120,30 @@ public Response getConfigs() {
         return delegate.getConfigs();
     }
 
+    @Valid
+    @GET
+    @Path("/homeRealmIdentifiers")
```

Comment:
home-realm-identifiers

Reply:
Addressed in https://github.com/wso2/identity-api-server/pull/260/commits/475f70efbd193afc7762fa8658b98f7cfedf290c